matic/contracts/WildcardSteward_matic_v2.sol

```diff
@@ -1244,4 +1244,50 @@ contract WildcardSteward_matic_v1 is Initializable, BasicMetaTransaction {
 
         assetToken.transferFrom(_currentOwner, address(this), tokenId);
     }
+
+    function transferWithDeposit(
+        address to,
+        uint256 tokenId,
+        uint256 newPrice,
+        uint256 tokensDeposit
+    )
+        public
+        collectPatronageAndSettleBenefactor(tokenId)
+        collectPatronagePatron(to)
+        priceGreaterThanZero(newPrice)
+        youCurrentlyAreNotInDefault(msgSender())
+        youCurrentlyAreNotInDefault(to) // Maybe this condition is too strict? Maybe it should check foreclosure time after recieving the token rather?
```

Comment:
Its fine I like this condition.

Reply:
If the new user is a brand new user, I am guessing it won't show them in default? Just checking